### ISSUE TITLE
Is there a way to configure this module to work with Bootstrap 3?

### ISSUE THREAD

**Issue by roman-kyrii** [mwillbanks/ZfcTwitterBootstrap#46]
Or this module works only with Bootstrap 2?

**Comment by trainerbill** [comment]
Yep... I also need 3.0 support

**Comment by str** [comment]
Doesn't this work on bootstrap 3?

**Comment by ThaDafinser** [comment]
@str @roman-kyrii @trainerbill use https://github.com/neilime/zf2-twb-bundle

This module seems to be dead...

**Comment by mwillbanks** [comment]
Indeed, this module is dead.  I've since been using a very simple implementation of bootstrap vs. bundling things together.
